### PR TITLE
Core: Add palash25

### PR DESCRIPTION
**Pull request by palash25**
Closes: https://github.com/coala/teams/issues/69

### CODE REVIEW

File: core/team-members.rst [hunk 1]
```diff
@@ -45,7 +45,12 @@ I am Hemang, I'm a Computer Science and Engineering undergrad, I like
 programming in Python, JavaScript and I also love making Android & Web projects.
 I like to learn new stuff everyday and to share my knowledge whenever I can.
 
-`Maxim Alexandru https://github.com/Nosferatul`_
+`Maxim Alexandru <https://github.com/Nosferatul>`_
```
Comment (Makman2) [inline core/team-members.rst:48]: heh unrelated change :P

**Review by Makman2 — CHANGES_REQUESTED** [review]

File: core/team-members.rst (hunk 1 above)
Comment (palash25) [reply core/team-members.rst:48]: His name was showing the full link to his profile unlike the rest and I'm a big fan of symmetry so couldn't resist :wink:

Comment (palash25) [reply core/team-members.rst:48]: is it okay if we keep or do you want me to amend the commit?

Comment (Makman2) [reply core/team-members.rst:48]: amend and new commit with this change

File: core/team-members.rst [hunk 2]
```diff
@@ -49,3 +49,8 @@ I like to learn new stuff everyday and to share my knowledge whenever I can.
 
 I'm Alex, undergraduate student from Romania. I love projects. I have contributed code to coala
 for almost 1 year. I'm helping newcomers and reviewing PRs and old Issues.
+
+
```
Comment (Makman2) [inline core/team-members.rst]: redundant newline ;)

File: core/team-members.rst (hunk 1 above)
Comment (palash25) [reply core/team-members.rst:48]: done @Makman2  :+1:

**Comment by Makman2** [comment]
ack 9b4e41a 12fa884

**Comment by Makman2** [comment]
@gitmate-bot ff

**Comment by Makman2** [comment]
Ah forgot gitmate isn't enabled here.
